### ISSUE TITLE
Changing learning rate

### ISSUE THREAD

**Issue by EYH0602** [EYH0602/ImageOrientation#1]
the training plot shows that the loss converges at a not good place. add functionality to change learning rate at this point.
